resources/asciidoctor/spec/elastic_include_tagged_spec.rb

```diff
@@ -118,8 +118,7 @@
     end
     it 'logs a warning about the missing tag' do
       expect(logs).to eq(
-        'WARN: <stdin>: line 5: elastic-include-tagged missing ' \
-        'start tag [missing_start]'
+        'WARN: <stdin>: line 5: missing start tag [missing_start]'
```

Comment:
The `elastic-include-tagged` bit in the error message wasn't really doing anyone any good and just made it longer.